libs/hpc-ui/src/lib/components/button.tsx

```diff
@@ -127,11 +130,22 @@ const StyledBaseButton = styled(BaseButton)`
   &:hover,
   &:focus,
   &.${CLS.ACTIVE} {
+    transition: ease-out 0.1s;
     text-decoration: none;
     background-color: ${(p) => p.theme.colors.pallete.gray.light};
     color: #fff;
   }
+  &.${COLOR_CLS.neutral_light} {
```

Comment:
This looks little out of place. You have added this definition where empty line was, which makes git report as single line was removed, while entire PR adds new code.

There is a certain order established in `COLOR_CLS` and it goes `primary`, `primary_light`, `secondary`, `secondary_light`, `neutral`, `neutral_light`, so I'd respect this order when using the variables and place `neutral_light` after `secondary_light`.

Not very important, but contributes to better readability.